addons/mail/models/res_users.py

```diff
@@ -128,6 +161,70 @@ def unlink(self):
         self._unsubscribe_from_non_public_channels()
         return super().unlink()
 
+    def _notify_security_setting_update(self, subject, text, suggest_2fa=True, suggest_password_reset=True, **kwargs):
```

Comment:
Thinking the "suggest" are not really part of the method ... Shouldn't we go for a preparation method that prepares some parameters based on the set of changes ? A tracking-like behavior ?